### PR TITLE
Labwhere Feb dep update

### PR DESCRIPTION
**Pull request by emrojo**
Closes #

Changes proposed in this pull request:

*
*
* ...

### CODE REVIEW

**Review by yoldas — APPROVED** [review]
I don't see anything that would be a concern.

File: app/controllers/location_types_controller.rb [hunk 1]
```diff
@@ -7,8 +7,11 @@ class LocationTypesController < ApplicationController
 
   def index; end
```
[the remark this answers is not in the document]
Comment (emrojo) [reply app/controllers/location_types_controller.rb:8]: I'm not sure, I think they can be removed but I would prefer doing it as part of another application refactor, as this pull request was only to upgrade libraries to be up to date with last security patches.

File: spec/rails_helper.rb [hunk 2]
```diff
@@ -27,7 +27,7 @@
   include FactoryBot::Syntax::Methods
 
   # Remove this line if you're not using ActiveRecord or ActiveRecord fixtures
-  config.fixture_path = "#{::Rails.root}/spec/fixtures"
+  config.fixture_path = Rails.root.join('/spec/fixtures')
```
Comment (yoldas) [inline spec/rails_helper.rb:30]: Do we need a forward slash before the segment "spec" or not?

Comment (emrojo) [reply spec/rails_helper.rb:30]: I think not, I'll remove it to leave it relative

**Pull request merged**